dags/common/utils.py

```diff
@@ -12,9 +13,14 @@ def set_harvesting_interval(repo, **kwargs):
             "until_date": kwargs["params"]["until_date"],
         }
     start_date = repo.find_the_last_uploaded_file_date()
-    until_date = date.today()
-
+    until_date = datetime.date.today().strftime("%Y-%m-%d")
```

Comment:
there is  a potential bug for `start_date` https://github.com/cern-sis/workflows/blob/e7d33f397a72df5abd87d8cd5bdb4ed7c7a30111/dags/aps/repository.py#L29 the first time we run it, if there are no s3 object it will return `None`

Reply:
no. it will put today's date